Sort/Program.cs

```diff
@@ -6,7 +6,7 @@ class Program
     {
         static void Main(string[] args)
         {
-
+            //Sorting numbers from smallest to largest//
             int[] numArray = { 23, 55, 99, 38, 65 };
 
             for (int i = 0; i < numArray.Length - 1; i++)
```

Comment:
This sort needs to work for any `int[]` I set this too. When you find yourself copying and pasting code to do something more than once that means you probably need a while loop. You need to be able to say, "sort this array of integers until all integers are sorted". Right now, if you attempt to sort the array `[5, 4, 3, 2, 1]` your output is not sorted.

Try to convert this into a loop that does just that, continue sorting until the array is sorted.